public_html/index.php

```diff
@@ -50,7 +50,7 @@ function injectScript( $scriptTag, $source ) {
 }
 
 function injectJQuery( $source ) {
-	return injectScript( '<script src="https://code.jquery.com/jquery-1.11.1.min.js"></script>', $source );
+	return injectScript( '<script src="https://tools-static.wmflabs.org/cdnjs/ajax/libs/jquery/1.11.1/jquery.min.js"></script>', $source );
```

Comment:
Confirmed identical:

```
$ sha256sum jq-*
540bc6dec1dd4b92ea4d3fb903f69eabf6d919afd48f4e312b163c28cff0f441  jq-cdn.111min.js
540bc6dec1dd4b92ea4d3fb903f69eabf6d919afd48f4e312b163c28cff0f441  jq-wmf.111min.js
```

Can you add `integrity="sha256-VAvG3sHdS5LqTT+5A/aeq/bZGa/Uj04xKxY8KM/w9EE="
  crossorigin="anonymous"` as well, per <https://code.jquery.com/jquery/> ?